src/mmw/js/src/core/tests.js

```diff
@@ -30,51 +30,55 @@ var chartData = [{x: 'a', y: 1},
                 {x: 'c', y: 3}],
     xValue = 'x',
     yValue = 'y',
-    sandboxHeight = '500',
-    sandboxWidth = '700',
-    sandboxSelector = '#display-sandbox';
+    displaySandboxHeight = '500',
+    displaySandboxWidth = '700',
+    displaySandboxSelector = '#display-sandbox',
+    sandboxId = 'sandbox',
+    sandboxSelector = '#' + sandboxId;
 
 describe('Core', function() {
     before(function() {
-        if ($('#sandbox').length === 0) {
-            $('<div>', {id: 'sandbox'}).appendTo('body');
+        if ($(sandboxSelector).length === 0) {
+            $('<div>', {id: sandboxId}).appendTo('body');
         }
     });
 
     beforeEach(function() {
-        $('#display-sandbox').remove();
+        $(displaySandboxSelector).remove();
         // Use a special sandbox so that we can test responsiveness of chart.
-        $('body').append(sandboxTemplate.render({height: sandboxHeight, width: sandboxWidth}));
+        $('body').append(sandboxTemplate.render({height: displaySandboxHeight, width: displaySandboxWidth}));
```

Comment:
This like got long. Consider replacing
`{height: displaySandboxHeight, width: displaySandboxWidth}`
with
```
{
    height: displaySandboxHeight, 
    width: displaySandboxWidth
}
```